appservice/src/createAppService/createWebApp.ts

```diff
@@ -23,23 +23,40 @@ export async function createWebApp(
     return await createAppService(AppKind.app, actionContext, subscriptionContext, createOptions, showCreatingTreeItem);
 }
 
-export async function setWizardContextDefaults(wizardContext: IAppServiceWizardContext): Promise<void> {
-    await LocationListStep.setLocation(wizardContext, 'centralus');
-    // defaults that for if one workspace is opened
+export async function getWizardRecommendations(wizardContext: IAppServiceWizardContext, actionContext: IActionContext): Promise<void> {
+    actionContext.properties.recommendedSiteRuntime = falseStr;
+    actionContext.properties.recommendedWebsiteOS = falseStr;
```

Comment:
Just default to undefined instead of using `falseStr`, which is a much more common pattern